src/Pages/Start/Start.css

```diff
@@ -0,0 +1,7 @@
+.start {
```

Comment:
Everything in the "Start" folder is setting up the page that you will see as soon as you've been authenticated (with the big Yellow button in the middle of it)